Don't include SHA of the HEAD of the aports repository in youki --version.

--- a/crates/youki/build.rs
+++ b/crates/youki/build.rs
@@ -3,6 +3,8 @@
 
 fn main() -> Result<()> {
     let mut config = Config::default();
-    *config.git_mut().sha_kind_mut() = ShaKind::Short;
+    // XXX-Patched: Disabled git output.
+    *config.git_mut().enabled_mut() = false;
+    println!("cargo:rustc-env=VERGEN_GIT_SHA_SHORT=0");
     vergen(config)
 }
